src/common/Utilities/Util.cpp

```diff
@@ -579,7 +579,7 @@ void HexStrToByteArray(std::string const& str, uint8* out, bool reverse /*= fals
 bool StringToBool(std::string const& str)
 {
     std::string lowerStr = str;
-    std::transform(str.begin(), str.end(), lowerStr.begin(), ::tolower);
+    std::transform(str.begin(), str.end(), lowerStr.begin(), [](char c) { return char(::tolower(c)); });
```

Comment:
But why

Reply:
Had an issue where the server would crash around here. Tried merging the change from: https://github.com/TrinityCore/TrinityCore/commit/4226eb2ea56a13f1e499b80e51cb73748cff82a1#diff-d66ed2e0118814a930a18391f6fb5ec5 but to no effect. Turns out it was caused by a typo I made. 
Left it in, since it would still fix a warning when compiling with /W4.